sim/core/spell.go

```diff
@@ -36,17 +36,16 @@ type SpellConfig struct {
 	Cast               CastConfig
 	ExtraCastCondition CanCastCondition
 
-	BonusHitRating       float64
-	BonusCritRating      float64
-	BonusSpellPower      float64
-	BonusExpertiseRating float64
-	BonusArmorPenRating  float64
+	BonusHitRating  float64
```

Comment:
Thanks for also removing BonusExpertiseRating and BonusArmorPenRating. Stats and PseudoStats _really_ need a cleanup